addons/web/controllers/main.py

```diff
@@ -762,7 +762,7 @@ def bootstrap_translations(self, mods):
         # For performance reasons we only load a single translation, so for
         # sub-languages (that should only be partially translated) we load the
         # main language PO instead - that should be enough for the login screen.
-        lang = request.lang.split('_')[0]
+        lang = request.context.get('lang', 'en_US').split('_')[0]
```

Comment:
Same as above, do not fallback on `en_US` as it may not be activated. Why isn't `request.lang` correct?

Reply:
`request.lang` does not exist at this point as it is not a `lazy_property` anymore, `lang` is added on `request` in `http_routing`.`_add_dispatch_parameters()` (that's the only place where `request.lang` was accessed outside `http_routing`).

Also, this controller is not called anymore, it is dead code.
Probably something to fix in lower version.
The JS supposed to fire the RPC is sending empty list of module as it try to get those from `window.odoo._modules`, which is not set anymore.

Session.init:
```javascript
this.module_list = (options.modules && options.modules.slice()) || (window.odoo._modules && window.odoo._modules.slice()) || [];
```
RPC:
```javascript
self.rpc('/web/webclient/bootstrap_translations', {mods: self.module_list})
```
and `options.modules` is also equivalent to `window.odoo._modules`, see
```javascript
var modules = odoo._modules;
var session = new Session(undefined, undefined, {modules: modules, use_cors: false});
```

Even tho, I made sure the controller was still working.
About the `en_US` fallback, I removed it, not needed, request.context.lang [will read](https://github.com/odoo/odoo/blame/2e640a37d45b53d28e5f5a67eede7855bb68a1d4/odoo/http.py#L253) request.session.context.lang which is set in [setup_lang](https://github.com/odoo/odoo/blame/2e640a37d45b53d28e5f5a67eede7855bb68a1d4/odoo/http.py#L1341)